src/common/iso_8601.cc

```diff
@@ -199,4 +199,54 @@ string to_iso_8601(const real_time t,
   out << '.' << setw(9) << nsec << 'Z';
   return out.str();
 }
+
+string to_iso_8601_no_separators(const real_time t,
+                                 const iso_8601_format f) noexcept {
+  ceph_assert(f >= iso_8601_format::Y &&
+	      f <= iso_8601_format::YMDhmsn);
+  stringstream out(std::ios_base::out);
```

Comment:
it's easy to replace later, but I think everything done here could be handled by fmt more efficiently

Reply:
yeah, I ended implementing casey's suggestion (to reuse `to_iso_8601()`)

Reply:
cool